### ISSUE TITLE
enforces-shorthand warning with responsive modifiers

### ISSUE THREAD

**Issue by fatcop** [francoismassart/eslint-plugin-tailwindcss#91]
From 3.1.0 onwards we are getting these warnings.

Have attached .eslintrc.js
[eslintrc.zip](https://github.com/francoismassart/eslint-plugin-tailwindcss/files/7836438/eslintrc.zip)

We are using `plugin:tailwindcss/recommended` and have ` groupByResponsive: false` (for classnames-order).

Getting warning `Classnames '' could be replaced by the 'py-2.5' shorthand! eslint (tailwindcss/enforces-shorthand)` for:
```html
<div className="py-2.5 md:py-3 pl-2.5 md:pl-4 font-medium uppercase">
```
- Removing `md:py-3` seems to fix it.

Getting warning `Classnames '' could be replaced by the 'md:px-10' shorthand! eslint (tailwindcss/enforces-shorthand)` for:
```html
<div className="px-6 md:px-10 pt-10 md:pt-16 pb-20 md:pb-10 bg-gray-100">
```
- Removing `md:px-10` seems to fix it.

Getting warning `Classnames '' could be replaced by the 'md:mx-6' shorthand! eslint (tailwindcss/enforces-shorthand)` for:
```html
<div className="lg:flex mx-4 md:mx-6 xl:mx-0 md:mt-6 lg:mt-0">
```
- Removing `xl:mx-0` seems to fix it.

Getting warning `Classnames '' could be replaced by the 'px-6' shorthand! eslint (tailwindcss/enforces-shorthand)` for:
```html
<div className="px-6 pt-5 pb-6 bg-indigo-700 rounded-lg">
```
- Removing `pb-6` seems to fix it, but not removing `pt-5` 🤷

**Comment by francoismassart** [comment]
@fatcop,
I had the same issue yesterday and the fix is ready, I’ll publish it right after dropping my son to school, in ~2 hours

**Comment by fatcop** [comment]
Legend. Thanks mate.

**Comment by francoismassart** [comment]
FIXED in `eslint-plugin-tailwindcss@3.1.2`
